Firestore/src/Query.php

```diff
@@ -306,40 +306,17 @@ public function select(array $fieldPaths)
     }
 
     /**
-     * Add a WHERE clause to the Query.
-     *
-     * For a list of all available operators, see
-     * {@see Google\Cloud\Firestore\V1\StructuredQuery\FieldFilter\Operator}.
-     * This method also supports a number of comparison operators which you will
-     * be familiar with, such as `=`, `>`, `<`, `<=` and `>=`. For array fields,
-     * the `array-contains`, `IN` and `array-contains-any` operators are also
-     * available.
-     *
-     * Example:
-     * ```
-     * $query = $query->where('firstName', '=', 'John');
-     * ```
+     * Create a non composite Filter.
      *
-     * ```
-     * // Filtering against `null` and `NAN` is supported only with the equality operator.
-     * $query = $query->where('coolnessPercentage', '=', NAN);
-     * ```
-     *
-     * ```
-     * // Use `array-contains` to select documents where the array contains given elements.
-     * $query = $query->where('friends', 'array-contains', ['Steve', 'Sarah']);
-     * ```
-     *
-     * @param string|FieldPath $fieldPath The field to filter by.
-     * @param string|int $operator The operator to filter by.
-     * @param mixed $value The value to compare to.
-     * @return Query A new instance of Query with the given changes applied.
+     * @param string|array|FieldPath $fieldPath A field to filter by.
+     * @param string|int $operator An operator to filter by.
+     * @param mixed $value A value to compare to.
+     * @return array A non composite Filter array.
      * @throws \InvalidArgumentException If an invalid operator or value is encountered.
      */
-    public function where($fieldPath, $operator, $value)
+    private function createNonCompositeFilter($fieldPath, $operator, $value)
```

Comment:
```suggestion
    private function createPropertyFilter($fieldPath, $operator, $value)
```

Reply:
property will be a new name which we are not using anywhere.
How about createSingleFieldFilter?
we will mean like a filter which operate on a single field. What do you think?

Reply:
changed name to `createSingleFieldFilter`